web/pandas_web.py

```diff
@@ -223,7 +224,23 @@ def home_add_releases(context):
         with open(pathlib.Path(context["target_path"]) / "releases.json", "w") as f:
             json.dump(releases, f, default=datetime.datetime.isoformat)
 
-        for release in releases:
+        # Map to create a list of (version, realse) pairs from releases
+        parsed_releases = map(lambda r: (version.parse(r["tag_name"]), r), releases)
+
+        # Sort the releases in descending order
+        sorted_releases = sorted(parsed_releases, reverse=True)
+
+        # Gathers minor versions that are NOT obsolete
```

Comment:
Maybe a short example of what we mean by obsolete, so future readers can understand it easier?